### ISSUE TITLE
about DUSt3R fine-tune

### ISSUE THREAD

**Issue by qiuzidian** [zhichu99/PanoSplatt3R#7]
I would like to ask whether you fine-tune DUSt3R (which has relative scale) directly using ground truth with metric scale?

**Comment by zhichu99** [comment]
We actually fine-tune MASt3R, not the original DUSt3R. MASt3R is already trained in metric scale,  and our fine-tuning is directly supervised with metric ground truth.
